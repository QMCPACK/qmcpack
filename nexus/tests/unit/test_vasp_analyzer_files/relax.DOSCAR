  13  13   0   0
  0.4096779E+02  0.5620240E-09  0.5620240E-09  0.1686072E-08  0.5000000E-15
   1.0000000000000000E-004
  CAR 
 unknown system                          
