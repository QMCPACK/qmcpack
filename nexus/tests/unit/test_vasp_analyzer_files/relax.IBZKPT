Automatically generated mesh
       3
Reciprocal lattice
    0.00000000000000    0.00000000000000    0.00000000000000             1
    0.33333333333333    0.00000000000000    0.00000000000000             4
    0.33333333333333    0.33333333333333    0.00000000000000             4
