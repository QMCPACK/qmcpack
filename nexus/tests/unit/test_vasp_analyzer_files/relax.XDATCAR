unknown system                          
            1
     5.620240    0.000000    0.000000
     0.000000    5.620240    0.000000
     0.000000    0.000000   16.860720
   Pt
  13
Direct configuration=     1
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23414099
   0.00000000  0.50000000  0.23441588
   0.50000000  0.00000000  0.23777213
   0.50000000  0.50000000  0.23414099
   0.25000000  0.25000000  0.35179823
   0.75000000  0.25000000  0.35179823
   0.25000000  0.75000000  0.35179823
   0.75000000  0.75000000  0.35179823
   0.00000000  0.50000000  0.44922707
Direct configuration=     2
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23322794
   0.00000000  0.50000000  0.23271048
   0.50000000  0.00000000  0.23863429
   0.50000000  0.50000000  0.23322794
   0.25564636  0.24435364  0.35088992
   0.74435364  0.24435364  0.35088992
   0.25564636  0.75564636  0.35088992
   0.74435364  0.75564636  0.35088992
   0.00000000  0.50000000  0.45106531
Direct configuration=     3
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23282440
   0.00000000  0.50000000  0.23261896
   0.50000000  0.00000000  0.23786719
   0.50000000  0.50000000  0.23282440
   0.25513996  0.24486004  0.35094343
   0.74486004  0.24486004  0.35094343
   0.25513996  0.75513996  0.35094343
   0.74486004  0.75513996  0.35094343
   0.00000000  0.50000000  0.44967788
Direct configuration=     4
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23253788
   0.00000000  0.50000000  0.23247082
   0.50000000  0.00000000  0.23752742
   0.50000000  0.50000000  0.23253788
   0.25556488  0.24443512  0.35060367
   0.74443512  0.24443512  0.35060367
   0.25556488  0.75556488  0.35060367
   0.74443512  0.75556488  0.35060367
   0.00000000  0.50000000  0.44891380
Direct configuration=     5
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23193200
   0.00000000  0.50000000  0.23216850
   0.50000000  0.00000000  0.23653208
   0.50000000  0.50000000  0.23193200
   0.25619668  0.24380332  0.34958766
   0.74380332  0.24380332  0.34958766
   0.25619668  0.75619668  0.34958766
   0.74380332  0.75619668  0.34958766
   0.00000000  0.50000000  0.44711331
Direct configuration=     6
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23138281
   0.00000000  0.50000000  0.23178925
   0.50000000  0.00000000  0.23556542
   0.50000000  0.50000000  0.23138281
   0.25654940  0.24345060  0.34847135
   0.74345060  0.24345060  0.34847135
   0.25654940  0.75654940  0.34847135
   0.74345060  0.75654940  0.34847135
   0.00000000  0.50000000  0.44561426
Direct configuration=     7
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23098498
   0.00000000  0.50000000  0.23140427
   0.50000000  0.00000000  0.23497072
   0.50000000  0.50000000  0.23098498
   0.25657175  0.24342825  0.34769376
   0.74342825  0.24342825  0.34769376
   0.25657175  0.75657175  0.34769376
   0.74342825  0.75657175  0.34769376
   0.00000000  0.50000000  0.44472603
Direct configuration=     8
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23078170
   0.00000000  0.50000000  0.23114241
   0.50000000  0.00000000  0.23477738
   0.50000000  0.50000000  0.23078170
   0.25650517  0.24349483  0.34732510
   0.74349483  0.24349483  0.34732510
   0.25650517  0.75650517  0.34732510
   0.74349483  0.75650517  0.34732510
   0.00000000  0.50000000  0.44440014
Direct configuration=     9
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23069404
   0.00000000  0.50000000  0.23100851
   0.50000000  0.00000000  0.23473405
   0.50000000  0.50000000  0.23069404
   0.25641397  0.24358603  0.34719108
   0.74358603  0.24358603  0.34719108
   0.25641397  0.75641397  0.34719108
   0.74358603  0.75641397  0.34719108
   0.00000000  0.50000000  0.44429967
Direct configuration=    10
   0.25000000  0.25000000  0.11785000
   0.75000000  0.25000000  0.11785000
   0.25000000  0.75000000  0.11785000
   0.75000000  0.75000000  0.11785000
   0.00000000  0.00000000  0.23068055
   0.00000000  0.50000000  0.23097015
   0.50000000  0.00000000  0.23475544
   0.50000000  0.50000000  0.23068055
   0.25638081  0.24361919  0.34716983
   0.74361919  0.24361919  0.34716983
   0.25638081  0.75638081  0.34716983
   0.74361919  0.75638081  0.34716983
   0.00000000  0.50000000  0.44431376
