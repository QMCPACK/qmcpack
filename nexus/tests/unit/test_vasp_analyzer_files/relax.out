Prologue Initiated
Creating global temporary directory /globalscratch/36892.b21l01.oic.ornl.gov
Creating local temporary directory /scratch/36892.b21l01.oic.ornl.gov
Running on node(s): b23n028
Prologue Complete

/home/j1k/projects/cnms/01_Cooper_high_throughput/06_neb_dev/02_relax_to_neb/runs/relax_hollow
 running on   32 total cores
 distrk:  each k-point on   32 cores,    1 groups
 distr:  one band on    1 cores,   32 groups
 using from now: INCAR     
 vasp.5.3.3 18Dez12 (build Dec 19 2012 13:15:22) complex                         
 POSCAR found type information on POSCAR  Pt
 POSCAR found :  1 types and      13 ions

 ----------------------------------------------------------------------------- 
|                                                                             |
|           W    W    AA    RRRRR   N    N  II  N    N   GGGG   !!!           |
|           W    W   A  A   R    R  NN   N  II  NN   N  G    G  !!!           |
|           W    W  A    A  R    R  N N  N  II  N N  N  G       !!!           |
|           W WW W  AAAAAA  RRRRR   N  N N  II  N  N N  G  GGG   !            |
|           WW  WW  A    A  R   R   N   NN  II  N   NN  G    G                |
|           W    W  A    A  R    R  N    N  II  N    N   GGGG   !!!           |
|                                                                             |
|      For optimal performance we recommend that you set                      |
|        NPAR = 4 - approx SQRT( number of cores)                             |
|      (number of cores/NPAR must be integer)                                 |
|      This setting will greatly improve the performance of VASP for DFT.     |
|      The default NPAR=number of cores might be grossly inefficient          |
|      on modern multi-core architectures or massively parallel machines.     |
|      Do your own testing.                                                   |
|      Unfortunately you need to use the default for hybrid, GW and RPA       |
|      calculations.                                                          |
|                                                                             |
 ----------------------------------------------------------------------------- 

 LDA part: xc-table for Pade appr. of Perdew
 POSCAR, INCAR and KPOINTS ok, starting setup
 WARNING: small aliasing (wrap around) errors must be expected
 FFT: planning ...
 WAVECAR not read
 entering main loop
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1     0.665011974615E+03    0.66501E+03   -0.36214E+04   576   0.622E+02 
DAV:   2    -0.450478175513E+02   -0.71006E+03   -0.63314E+03   672   0.270E+02 
DAV:   3    -0.755979514099E+02   -0.30550E+02   -0.29565E+02   768   0.552E+01 
DAV:   4    -0.760236542117E+02   -0.42570E+00   -0.42556E+00   672   0.688E+00 
DAV:   5    -0.760344123165E+02   -0.10758E-01   -0.10757E-01   864   0.948E-01    0.134E+01
DAV:   6    -0.734137765864E+02    0.26206E+01   -0.15347E+01   704   0.207E+01    0.104E+01
DAV:   7    -0.753367889759E+02   -0.19230E+01   -0.20145E+01   800   0.293E+01    0.126E+01
DAV:   8    -0.728974920476E+02    0.24393E+01   -0.10013E+01   864   0.189E+01    0.854E+00
DAV:   9    -0.714007699908E+02    0.14967E+01   -0.24275E+00   832   0.112E+01    0.260E+00
DAV:  10    -0.713354493582E+02    0.65321E-01   -0.26930E-01   704   0.237E+00    0.142E+00
DAV:  11    -0.713274691716E+02    0.79802E-02   -0.11245E-01   896   0.180E+00    0.101E+00
DAV:  12    -0.713210758729E+02    0.63933E-02   -0.67965E-02   736   0.127E+00    0.552E-01
DAV:  13    -0.713185276012E+02    0.25483E-02   -0.26293E-02   864   0.874E-01    0.307E-01
DAV:  14    -0.713177956235E+02    0.73198E-03   -0.64036E-03   800   0.373E-01    0.103E-01
DAV:  15    -0.713177487485E+02    0.46875E-04   -0.81584E-04   832   0.176E-01    0.533E-02
DAV:  16    -0.713177334238E+02    0.15325E-04   -0.23721E-04   608   0.709E-02    0.268E-02
DAV:  17    -0.713178580292E+02   -0.12461E-03   -0.91300E-05   672   0.438E-02    0.189E-02
DAV:  18    -0.713181186840E+02   -0.26065E-03   -0.54501E-05   640   0.190E-02    0.124E-02
DAV:  19    -0.713182760933E+02   -0.15741E-03   -0.11447E-05   704   0.146E-02    0.468E-03
DAV:  20    -0.713184071630E+02   -0.13107E-03   -0.58661E-06   640   0.820E-03    0.409E-03
DAV:  21    -0.713184703206E+02   -0.63158E-04   -0.17863E-06   640   0.484E-03    0.164E-03
DAV:  22    -0.713184853424E+02   -0.15022E-04   -0.48009E-07   448   0.246E-03    0.664E-04
DAV:  23    -0.713184889218E+02   -0.35794E-05   -0.21068E-07   544   0.142E-03    0.409E-04
DAV:  24    -0.713184899385E+02   -0.10167E-05   -0.69977E-08   512   0.827E-04    0.180E-04
DAV:  25    -0.713184902339E+02   -0.29544E-06   -0.29187E-08   448   0.431E-04 
   1 F= -.71318490E+02 E0= -.71299119E+02  d E =-.713185E+02 
 BRION: g(F)=  0.238E+00 g(S)=  0.000E+00 
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.713280684777E+02   -0.95785E-02   -0.69316E+01   576   0.207E+01    0.104E+00
DAV:   2    -0.715227268985E+02   -0.19466E+00   -0.15006E+00   832   0.459E+00    0.247E+00
DAV:   3    -0.714309202939E+02    0.91807E-01   -0.47644E-01   736   0.369E+00    0.110E+00
DAV:   4    -0.714189489628E+02    0.11971E-01   -0.61475E-02   672   0.137E+00    0.488E-01
DAV:   5    -0.714161910564E+02    0.27579E-02   -0.75421E-03   672   0.483E-01    0.369E-01
DAV:   6    -0.714167933880E+02   -0.60233E-03   -0.96402E-03   640   0.437E-01    0.260E-01
DAV:   7    -0.714165352787E+02    0.25811E-03   -0.60164E-03   832   0.441E-01    0.942E-02
DAV:   8    -0.714167244329E+02   -0.18915E-03   -0.19581E-03   672   0.202E-01    0.105E-01
DAV:   9    -0.714165524663E+02    0.17197E-03   -0.67713E-04   768   0.126E-01    0.252E-02
DAV:  10    -0.714165615715E+02   -0.91052E-05   -0.79320E-05   576   0.290E-02    0.106E-02
DAV:  11    -0.714165618104E+02   -0.23888E-06   -0.87386E-06   832   0.153E-02 
   2 F= -.71416562E+02 E0= -.71399519E+02  d E =-.980716E-01 
 BRION: g(F)=  0.278E-01 g(S)=  0.000E+00 retain N=  1 mean eig= 0.88
 eig:   0.885
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714271123813E+02   -0.10551E-01   -0.53130E+00   576   0.549E+00    0.360E-01
DAV:   2    -0.714458276021E+02   -0.18715E-01   -0.12696E-01   832   0.166E+00    0.877E-01
DAV:   3    -0.714331027780E+02    0.12725E-01   -0.48122E-02   768   0.123E+00    0.390E-01
DAV:   4    -0.714313151961E+02    0.17876E-02   -0.90457E-03   736   0.536E-01    0.133E-01
DAV:   5    -0.714312625363E+02    0.52660E-04   -0.67211E-04   800   0.128E-01    0.933E-02
DAV:   6    -0.714314465449E+02   -0.18401E-03   -0.59455E-04   672   0.124E-01    0.714E-02
DAV:   7    -0.714315232424E+02   -0.76697E-04   -0.54071E-04   864   0.119E-01    0.373E-02
DAV:   8    -0.714315120883E+02    0.11154E-04   -0.15069E-04   768   0.561E-02    0.182E-02
DAV:   9    -0.714315116354E+02    0.45288E-06   -0.21855E-05   768   0.195E-02    0.797E-03
DAV:  10    -0.714315106807E+02    0.95469E-06   -0.59039E-06   736   0.108E-02 
   3 F= -.71431511E+02 E0= -.71414184E+02  d E =-.149489E-01 
 BRION: g(F)=  0.798E-02 g(S)=  0.000E+00 retain N=  1 mean eig= 0.60
 eig:   0.604
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714363416931E+02   -0.48301E-02   -0.24988E+00   576   0.386E+00    0.154E-01
DAV:   2    -0.714411583562E+02   -0.48167E-02   -0.40283E-02   800   0.798E-01    0.347E-01
DAV:   3    -0.714391451693E+02    0.20132E-02   -0.71343E-03   704   0.455E-01    0.141E-01
DAV:   4    -0.714390023516E+02    0.14282E-03   -0.86173E-04   768   0.166E-01    0.866E-02
DAV:   5    -0.714390177468E+02   -0.15395E-04   -0.16852E-04   736   0.799E-02    0.702E-02
DAV:   6    -0.714390990250E+02   -0.81278E-04   -0.51723E-04   640   0.853E-02    0.311E-02
DAV:   7    -0.714390956155E+02    0.34095E-05   -0.80823E-05   928   0.620E-02    0.116E-02
DAV:   8    -0.714391001628E+02   -0.45473E-05   -0.22667E-05   736   0.220E-02    0.137E-02
DAV:   9    -0.714390976070E+02    0.25558E-05   -0.11043E-05   608   0.157E-02    0.524E-03
DAV:  10    -0.714390980322E+02   -0.42515E-06   -0.18115E-06   480   0.420E-03 
   4 F= -.71439098E+02 E0= -.71422214E+02  d E =-.758735E-02 
 BRION: g(F)=  0.572E-02 g(S)=  0.000E+00 retain N=  2 mean eig= 2.99
 eig:   0.572  5.402
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714348199276E+02    0.42777E-02   -0.15862E+01   576   0.977E+00    0.394E-01
DAV:   2    -0.714685880170E+02   -0.33768E-01   -0.27706E-01   896   0.220E+00    0.938E-01
DAV:   3    -0.714544035249E+02    0.14184E-01   -0.47806E-02   800   0.120E+00    0.341E-01
DAV:   4    -0.714532079992E+02    0.11955E-02   -0.47540E-03   736   0.382E-01    0.224E-01
DAV:   5    -0.714532235903E+02   -0.15591E-04   -0.10671E-03   704   0.210E-01    0.189E-01
DAV:   6    -0.714533577796E+02   -0.13419E-03   -0.39227E-03   736   0.219E-01    0.561E-02
DAV:   7    -0.714533700836E+02   -0.12304E-04   -0.31490E-04   960   0.126E-01    0.291E-02
DAV:   8    -0.714533828769E+02   -0.12793E-04   -0.77925E-05   800   0.451E-02    0.256E-02
DAV:   9    -0.714533745610E+02    0.83159E-05   -0.44741E-05   864   0.313E-02    0.140E-02
DAV:  10    -0.714533722811E+02    0.22799E-05   -0.10659E-05   640   0.118E-02    0.625E-03
DAV:  11    -0.714533734059E+02   -0.11249E-05   -0.20959E-06   544   0.742E-03    0.407E-03
DAV:  12    -0.714533755296E+02   -0.21236E-05   -0.31311E-06   640   0.642E-03    0.111E-03
DAV:  13    -0.714533771756E+02   -0.16461E-05   -0.16382E-07   480   0.151E-03    0.692E-04
DAV:  14    -0.714533804398E+02   -0.32641E-05   -0.23431E-07   480   0.118E-03    0.653E-04
DAV:  15    -0.714533815590E+02   -0.11192E-05   -0.59948E-08   480   0.767E-04    0.309E-04
DAV:  16    -0.714533821946E+02   -0.63564E-06   -0.48708E-08   416   0.568E-04 
   5 F= -.71453382E+02 E0= -.71437599E+02  d E =-.142842E-01 
 BRION: g(F)=  0.252E-02 g(S)=  0.000E+00 retain N=  1 mean eig= 4.25
 eig:   4.253
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714441283064E+02    0.92533E-02   -0.15025E+01   576   0.953E+00    0.345E-01
DAV:   2    -0.714711105316E+02   -0.26982E-01   -0.23676E-01   896   0.194E+00    0.750E-01
DAV:   3    -0.714621697453E+02    0.89408E-02   -0.30148E-02   832   0.963E-01    0.302E-01
DAV:   4    -0.714612607100E+02    0.90904E-03   -0.29034E-03   640   0.275E-01    0.217E-01
DAV:   5    -0.714613434704E+02   -0.82760E-04   -0.12078E-03   768   0.221E-01    0.191E-01
DAV:   6    -0.714612894305E+02    0.54040E-04   -0.39205E-03   800   0.226E-01    0.293E-02
DAV:   7    -0.714613570078E+02   -0.67577E-04   -0.19530E-04   768   0.675E-02    0.357E-02
DAV:   8    -0.714613426745E+02    0.14333E-04   -0.95877E-05   800   0.491E-02    0.187E-02
DAV:   9    -0.714613380053E+02    0.46692E-05   -0.19227E-05   736   0.192E-02    0.108E-02
DAV:  10    -0.714613360874E+02    0.19178E-05   -0.29161E-06   576   0.113E-02    0.730E-03
DAV:  11    -0.714613354995E+02    0.58788E-06   -0.73966E-06   608   0.802E-03 
   6 F= -.71461335E+02 E0= -.71446666E+02  d E =-.795330E-02 
 BRION: g(F)=  0.806E-03 g(S)=  0.000E+00 retain N=  1 mean eig= 6.08
 eig:   6.082
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714562406213E+02    0.50955E-02   -0.67209E+00   576   0.638E+00    0.208E-01
DAV:   2    -0.714663200583E+02   -0.10079E-01   -0.93954E-02   768   0.114E+00    0.389E-01
DAV:   3    -0.714639319970E+02    0.23881E-02   -0.87240E-03   736   0.497E-01    0.194E-01
DAV:   4    -0.714636100759E+02    0.32192E-03   -0.99792E-04   640   0.163E-01    0.147E-01
DAV:   5    -0.714636673566E+02   -0.57281E-04   -0.57771E-04   800   0.150E-01    0.130E-01
DAV:   6    -0.714636286977E+02    0.38659E-04   -0.17360E-03   736   0.162E-01    0.270E-02
DAV:   7    -0.714636690674E+02   -0.40370E-04   -0.18934E-04   832   0.652E-02    0.333E-02
DAV:   8    -0.714636569544E+02    0.12113E-04   -0.72063E-05   800   0.394E-02    0.140E-02
DAV:   9    -0.714636543182E+02    0.26362E-05   -0.94859E-06   672   0.145E-02    0.885E-03
DAV:  10    -0.714636528683E+02    0.14499E-05   -0.18440E-06   544   0.890E-03    0.533E-03
DAV:  11    -0.714636528748E+02   -0.65302E-08   -0.50850E-06   640   0.623E-03 
   7 F= -.71463653E+02 E0= -.71449717E+02  d E =-.231738E-02 
 BRION: g(F)=  0.219E-03 g(S)=  0.000E+00 retain N=  1 mean eig= 6.99
 eig:   6.989
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714624393350E+02    0.12135E-02   -0.14086E+00   576   0.292E+00    0.865E-02
DAV:   2    -0.714642179570E+02   -0.17786E-02   -0.17744E-02   736   0.451E-01    0.122E-01
DAV:   3    -0.714639993944E+02    0.21856E-03   -0.91473E-04   832   0.154E-01    0.870E-02
DAV:   4    -0.714639401433E+02    0.59251E-04   -0.17705E-04   704   0.711E-02    0.684E-02
DAV:   5    -0.714639568287E+02   -0.16685E-04   -0.18060E-04   768   0.744E-02    0.615E-02
DAV:   6    -0.714639387498E+02    0.18079E-04   -0.34320E-04   736   0.799E-02    0.139E-02
DAV:   7    -0.714639497091E+02   -0.10959E-04   -0.47402E-05   864   0.322E-02    0.153E-02
DAV:   8    -0.714639468786E+02    0.28305E-05   -0.14748E-05   768   0.192E-02    0.737E-03
DAV:   9    -0.714639462310E+02    0.64752E-06   -0.15658E-06   384   0.781E-03 
   8 F= -.71463946E+02 E0= -.71450340E+02  d E =-.293356E-03 
 BRION: g(F)=  0.585E-04 g(S)=  0.000E+00 retain N=  1 mean eig= 6.09
 eig:   6.088
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714637032775E+02    0.24360E-03   -0.20862E-01   576   0.112E+00    0.339E-02
DAV:   2    -0.714639506814E+02   -0.24740E-03   -0.25176E-03   608   0.165E-01    0.390E-02
DAV:   3    -0.714639323967E+02    0.18285E-04   -0.14155E-04   704   0.535E-02    0.330E-02
DAV:   4    -0.714639261653E+02    0.62314E-05   -0.36010E-05   640   0.343E-02    0.260E-02
DAV:   5    -0.714639260433E+02    0.12196E-06   -0.18134E-05   768   0.244E-02    0.232E-02
DAV:   6    -0.714639257763E+02    0.26708E-06   -0.49687E-05   736   0.263E-02    0.673E-03
DAV:   7    -0.714639286595E+02   -0.28833E-05   -0.10894E-05   832   0.154E-02    0.677E-03
DAV:   8    -0.714639283512E+02    0.30836E-06   -0.25089E-06   384   0.902E-03 
   9 F= -.71463928E+02 E0= -.71450426E+02  d E =0.178799E-04 
 BRION: g(F)=  0.188E-04 g(S)=  0.000E+00 retain N=  1 mean eig= 2.33
 eig:   2.330
 bond charge predicted
       N       E                     dE             d eps       ncg     rms          rms(c)
DAV:   1    -0.714639178246E+02    0.10835E-04   -0.87899E-03   576   0.231E-01    0.904E-03
DAV:   2    -0.714639301023E+02   -0.12278E-04   -0.11799E-04   608   0.381E-02    0.136E-02
DAV:   3    -0.714639278739E+02    0.22284E-05   -0.16273E-05   768   0.225E-02    0.862E-03
DAV:   4    -0.714639270439E+02    0.82998E-06   -0.34097E-06   384   0.101E-02 
  10 F= -.71463927E+02 E0= -.71450439E+02  d E =0.130726E-05 
 BRION: g(F)=  0.113E-04 g(S)=  0.000E+00 retain N=  2 mean eig= 4.80
 eig:   2.630  6.975
 reached required accuracy - stopping structural energy minimisation
 writing wavefunctions
Epilogue Initiated
Removing global temporary directory /globalscratch/36892.b21l01.oic.ornl.gov
Removing local temporary directory /scratch/36892.b21l01.oic.ornl.gov
Epilogue Complete
